README.md

```diff
@@ -7,6 +7,35 @@ The IBM Blockchain extension allows a user to write and test chaincode projects
 * Generate skeleton chaincode projects
 * Adds a Fabric view to connect and interact with Fabric Blockchain networks
 
+
+## Requirements
+
+The extension provides functionality to connect to a Hyperledger Fabric instance to discover existing resources, install chaincode and instantiate chaincode.
```

Comment:
Change to:

"Using this extension, you can connect to a pre-configured local instance of Hyperledger Fabric, or you can connect to your own Hyperledger Fabric instance. If you choose to connect to your own Fabric instance, it must be Fabric 1.3.0 or later."